src/modules/dashboard/lists/SuppliedPositionsList/SuppliedPositionsList.tsx

```diff
@@ -41,16 +46,90 @@ export const SuppliedPositionsList = () => {
       })) || [];
 
   const head = [
-    <Trans key="Balance">Balance</Trans>,
-    <Trans key="APY">APY</Trans>,
-    <CollateralSwitchTooltip
-      text={<Trans>Collateral</Trans>}
-      key="Collateral"
-      variant="subheader2"
-    />,
+    {
+      title: <Trans>Asset</Trans>,
+      sortKey: 'symbol',
+    },
+    {
+      title: <Trans key="Balance">Balance</Trans>,
+      sortKey: 'underlyingBalance',
+    },
+
+    {
+      title: <Trans key="APY">APY</Trans>,
+      sortKey: 'supplyAPY',
+    },
+    {
+      title: (
+        <CollateralSwitchTooltip
+          text={<Trans>Collateral</Trans>}
+          key="Collateral"
+          variant="subheader2"
+        />
+      ),
+      sortKey: 'usageAsCollateralEnabledOnUser',
+    },
   ];
 
-  if (loading) return <ListLoader title={<Trans>Your supplies</Trans>} head={head} />;
+  if (sortDesc) {
+    if (sortName === 'symbol') {
+      suppliedPosition.sort((a, b) =>
+        a.reserve.symbol.toUpperCase() < b.reserve.symbol.toUpperCase() ? -1 : 1
+      );
+    } else if (sortName === 'usageAsCollateralEnabledOnUser') {
+      suppliedPosition.sort(
+        (a, b) =>
+          Number(a.usageAsCollateralEnabledOnUser) - Number(b.usageAsCollateralEnabledOnUser)
+      );
+    } else {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      suppliedPosition.sort((a, b) => a[sortName] - b[sortName]);
+    }
+  } else {
+    if (sortName === 'symbol') {
+      suppliedPosition.sort((a, b) =>
+        b.reserve.symbol.toUpperCase() < a.reserve.symbol.toUpperCase() ? -1 : 1
+      );
+    } else if (sortName === 'usageAsCollateralEnabledOnUser') {
+      suppliedPosition.sort(
+        (a, b) =>
+          Number(b.usageAsCollateralEnabledOnUser) - Number(a.usageAsCollateralEnabledOnUser)
+      );
+    } else {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      suppliedPosition.sort((a, b) => b[sortName] - a[sortName]);
+    }
+  }
+
+  const renderHeader = () => {
+    return (
+      <ListHeaderWrapper>
+        {head.map((col) => (
+          <ListColumn
+            isRow={col.sortKey === 'symbol'}
+            maxWidth={col.sortKey === 'symbol' ? 160 : undefined}
+            key={col.sortKey}
+          >
+            <ListHeaderTitle
+              sortName={sortName}
+              sortDesc={sortDesc}
+              setSortName={setSortName}
+              setSortDesc={setSortDesc}
+              sortKey={col.sortKey}
+            >
+              {col.title}
+            </ListHeaderTitle>
+          </ListColumn>
+        ))}
+        <ListColumn maxWidth={170} minWidth={170} />
+      </ListHeaderWrapper>
+    );
+  };
```

Comment:
Similar comments for all these lists on the header SFC

Reply:
3b6ecbb6ead3f77234fd8f05c8749bbf5da69a4d